spec/dummy/app/services/custom_parser.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class CustomParser < Metka::GenericParser
```

Comment:
Maybe we can name it a bit more specificaly?

Reply:
I don't know, it's just a Parser for tests, we could call it TestParser, to be more specific with it's purpose, but it's probably also wouldn't be entirely true, since we would still generally be using GenericParser

Reply:
We could probably could call it `PipeParser`, that would highlight the fact, that it's using pipe delimiter, but I think, that calling it `CustomParser` to antagonize it to `GenericParser` is OK in that case

Reply:
Ah, it's only for tests, then there is no problem